proposals/Exceptions.md

```diff
@@ -240,19 +234,21 @@ If the `kind` is `Exception`:
 
 | Field | Type | Description |
 |-------|------|-------------|
-| type  | `varuint32` | exception index to table in the *exception type section* |
+| tag   | varuint32 | index into exception section |
```

Comment:
I think I understand now. Brad told me about function index spaces. If we use a similar concept for an "Exception index space", where imports appear first, we can remove the tag from the import.